src/main/resources/META-INF/native-image/io.lettuce/lettuce-core/reflect-config.json

```diff
@@ -169,6 +169,16 @@
     "allPublicMethods": true,
     "allDeclaredMethods": true
   },
+  {
+    "name": "io.lettuce.core.pubsub.RedisPubSubAsyncCommandsImpl",
+    "allPublicMethods": true,
+    "allDeclaredConstructors": true
+  },
+  {
+    "name": "io.lettuce.core.api.sync.BaseRedisCommands",
```

Comment:
This one duplicates lines 37 to 41. Can I drop that during the merge to avoid duplicates?

Reply:
Yes, sorry about that.